### DISCUSSION TITLE
piVPN: Changing IP from Reserved to Static

### DISCUSSION THREAD

**Discussion by wantle** [pivpn/pivpn#1439] [Support]
Hi,

I need to change my pivpn from a reserved IP to static IP (no IP change).

Can someone tell me how to accomplish this change? If I'm keeping the same IP parameters, should I avoid any issues? I also have pi-hole installed. pivpn is using Wireguard

Thanks

/W

**Comment by coolapso** [comment]
Hi, 

Looking at what you are describing you should have no issues. 

as for how to do it ... there are different ways to do it .. either to make a static IP reservation on your dhcp server (typically the router) and it will always provide the same IP address to your PI ... or you set it up on `/etc/dhcpcd.conf` ... just google "raspbian static ip address" and you will get plenty of results. 

I personally prefer to use static IP reservations on the DHCP server.
